src/dbnode/generated/proto/schema_test/test1.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package schematest;
+
+message TestMessage {
```

Comment:
Could you throw in some repeated fields, a nested message, and a map field as well?

Reply:
ack